commands.py

```diff
@@ -0,0 +1,32 @@
+from LSP.plugin.core.protocol import Location, LocationLink
+from LSP.plugin.core.typing import List, Union
+from LSP.plugin.execute_command import LspExecuteCommand
+from LSP.plugin.locationpicker import LocationPicker
+
+
+SESSION_NAME = __package__
+
+
+class LspTypescriptExecuteCommand(LspExecuteCommand):
```

Comment:
I don't think this is a good name, there is already a command with the same name in the LSP-typescript package: https://github.com/sublimelsp/LSP-typescript/blob/2569d61cf0a913de8e52d3738315cb6579f930af/commands.py#L10

I'm not a TypeScript user and don't know whether there are users who would install both packages at the same time. But this looks like a potential source of bugs from having two packages that provide commands with the same name. You should probably use a uniqe name like LspVtslsExecuteCommand

Reply:
Technically user should not install both plugins at a same time cause both doing same job. but still you are right. 

i have changed the name to `LspVtslsExecuteCommand`

Reply:
You still need to update the corresponding command name in the `.sublime-commands` file (`lsp_vtsls_execute`).

By the way, the other command LspTypescriptGotoSourceDefinitionCommand has also the same name as the one from the LSP-typescript package.